pkg/services/ngalert/models/testing.go

```diff
@@ -140,6 +140,21 @@ func WithOrgID(orgId int64) AlertRuleMutator {
 	}
 }
 
+func WithUniqueOrgID() AlertRuleMutator {
```

Comment:
In the past I've used `sync/atomic` to keep incrementing a uint64, avoids the need to keep a map of orgIDs.

Reply:
I chose this approach because of two reasons:
1. there is a similar function WithUniqueID that implements the same approach
2. it lets us use randomization instead of monotonic increments, which expands the test to all possible values of the field type.